### ISSUE TITLE
Image path change in Essential Graphics Template

### ISSUE THREAD

**Issue by AkiLetschne** [qmasingarbe/pymiere#76]
Hey, I have created an essential graphics template with aftereffects, where there is an image that is supposed to be a placeholder. It then has an attribute on the template itself. This attribute also appears in the properties, but the value of it is only many zeros asa value (00000000-0000-0000-0000-0000-000000000000). Is there any way to change the path with pymiere?

Thanks for your help and props for the nice library :)

**Comment by warped-kings** [comment]
I don't believe it's possible. I've tried a few different things through code but couldn't get anything work.

If you take the .mogrt, change the extension to .zip and open it, you'll see a json file with all the control items.
For an image, you might see the cacheID as all zeros like you mentioned. Then under thumbnail you'll find something like "c8f871a9-8b9f-4742-8a39-4f875d2404bf.png".

**Comment by sylvqin** [comment]
Anyone got any idea about this? Tried as well and couldn't get it to work, this would be super useful!
Thanks.
